monai/engines/evaluator.py

```diff
@@ -59,7 +59,7 @@ def __init__(
         post_transform: Optional[Transform] = None,
         key_val_metric: Optional[Dict[str, Metric]] = None,
         additional_metrics: Optional[Dict[str, Metric]] = None,
-        val_handlers: Optional[Sequence] = None,
+        val_handlers: Optional[Sequence[Metric]] = None,
```

Comment:
val_handler is not `Metric`, it's MONAI own classes under `handlers/`, basically, it can be any object that can handle data, so I just put `Sequence` for it.

Reply:
My bad, I was constraining it for this call. I should look into some sort of duck typing solution.

```python
handler.attach(self)
```